open_schools_platform/history_management/serializers/circle_serializer.py

```diff
@@ -0,0 +1,20 @@
+from rest_framework import serializers
+
+from open_schools_platform.organization_management.circles.models import Circle
+
+
+class HistoryRecordsField(serializers.ListField):
+    child = serializers.DictField()
+
+    def to_representation(self, data):
+        return super().to_representation(data.values("history_id", "history_user_id", "history_date", "history_type",
+                                                     "id", "name", "organization_id", "description", "capacity",
+                                                     "address",))
+
+
+class CircleHistorySerializer(serializers.ModelSerializer):
+    history = HistoryRecordsField(read_only=True)
+
+    class Meta:
+        model = Circle
+        fields = ("history",)
```

Comment:
it would be great to reduce the amount of code for serializers
think how to do it